src/main/scala/io/github/gaelrenoux/tranzactio/doobie/package.scala

```diff
@@ -27,13 +27,13 @@ package object doobie extends Wrapper {
   override final def tzio[A](q: => Query[A])(implicit trace: Trace): TranzactIO[A] =
     ZIO.serviceWithZIO[Connection] { c =>
       c.trans.apply(q)
-    }.mapError(DbException.Wrapped)
+    }.mapError(DbException.Wrapped.apply)
 
   /** Converts a Doobie stream to a ZStream. Note that you can provide a queue size, default value is the same as in ZIO. */
   final def tzioStream[A](q: => fs2.Stream[Query, A], queueSize: => Int = DefaultStreamQueueSize)(implicit trace: Trace): TranzactIOStream[A] =
     ZStream.serviceWithStream[Connection] { c =>
       c.transP.apply(q).toZStream(queueSize)
-    }.mapError(DbException.Wrapped)
+    }.mapError(DbException.Wrapped.apply)
```

Comment:
There were warnings from compiler suggesting me to explicit call `apply` methods so I did.